DirectusExtensions/charging-stations-bundle/src/display-true-count/display.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+	<span>{{ calculatedValue }}{{ totalPrefix }}{{ total }}{{ suffix }}</span>
+</template>
+
+<script lang="ts">
+// Copyright Contributors to the CitrineOS Project
+//
+// SPDX-License-Identifier: Apache 2.0
```

Comment:
Very minor: how about moving this part to the top?

Reply:
prevents the code from compiling, unfortunately.